drivers/enc28j60/enc28j60.c

```diff
@@ -248,6 +256,24 @@ static int nd_send(netdev_t *netdev, const iolist_t *iolist)
 
     mutex_lock(&dev->devlock);
 
+    if (cmd_rcr(dev, REG_ECON1, -1) & ECON1_TXRTS) {
+        /* there is already a transmission in progress */
+        if (xtimer_now_usec() - dev->tx_time > MAX_TX_TIME * 2) {
+            /* if transmission time exceeds the double of maximum transmission
+               time, we suppose that TX logic hangs and has to be reset */
```

Comment:
here and below: please uses `*` at the beginning of subsequent lines in multi-line comments